micb_soa.m

```diff
@@ -240,20 +241,49 @@
             timesUp = 1;
         end
     end
-    
+    %%% Create a 4X8 double 
+    non_targets = centredRects(1:8);
+    %%% Reduce to a 4X7 by removing the target
+    non_targets(:,target) = [];
+    incorrectRect = centeredRects(:,non_targets);
+    incorrectRect = incorrectRect;
     correctRect = centeredRects(:,target);
     correctRect = correctRect';
     
+    %%% KeySet for all dictionaries
+    incorrect_keySet = {'1','2','3','4','5','6','7'};
+    %%% X axis value keys
+    incorrect_valueSet_left = {non_targets(1,:)}; incorrect_valueSet_right = {non_targets(3,:)};
+    %%% Y axis value keys
+    incorrect_valueSet_bottom = {non_targets(2,:)}; incorrect_valueSet_top = {non_targets(4,:)};
+    %%% X axis map
+    incorrect_Map_left = containers.Map(incorrect_keySet,valueSet_time_1); incorrect_Map_right = containers.Map(incorrect_keySet,valueSet_time_2);
+    %%% Y axis map
+    incorrect_Map_top = containers.Map(incorrect_keySet,valueSet_time_2); incorrect_Map_bottom = containers.Map(incorrect_keySet,valueSet_time_2);
+    
     if clicked==1&&x>=correctRect(1)&&x<=correctRect(3)&&y>=correctRect(2)&&y<=correctRect(4)
         accuracy = 1;
         RT = GetSecs-startTime;
         Screen('FillRect',w,bgcolor,rect);
         DrawFormattedText(w,'Correct','center','center');
         Screen('Flip',w);
     elseif clicked==1
+        for i = length(incorrect_keySet)
+            if x>=incorrect_Map_left(i)&&x<=incorrect_Map_right(i)&&y>=incorrect_Map_top(i)&&y<=incorrect_Map_bottom(i)
+                incorrect_gabor = i

```

Comment:
I assume you are still working on adding this extra stuff to the output file you save, 
are you saving a .mat file or a .csv file with the results for each participant?

Reply:
It is a .mat file. We have the final version of the output stuff that will be saved at the end of the experiment.